frontend/src/pages/CanITeachPage/CanITeachPage.tsx

```diff
@@ -81,7 +82,14 @@ const CanITeachPage: React.FC = () => (
         <strong>výkladu tohoto zákona, který jsme pro vás připravili</strong>.
       </Paragraph>
       <SectionSubtitle>Rozcestník: kde chcete učit?</SectionSubtitle>
-      <Paragraph>Vyberte si, kde chcete učit, a my vám ukážeme, jak na to.</Paragraph>
+      <Paragraph>
+        Vyberte si, kde chcete učit, a my vám ukážeme, jak na to. Pokud si nejste jistí, jaké
+        předměty se svým vzděláním můžete vyučovat, podívejte se do naší{' '}
+        <MainLink href="./subjects.pdf" target="_blank" download>
```

Comment:
Nestačilo by sem dát `<Button buttonStyle="link" variant="primary">` nebo `<Button buttonStyle="link" variant="secondary">`? 

Třeba ne a má ten link vypadat jinak, ale styly pro link jsou nadefinovaný už v tý button komponentě tak to třeba pujde